tests/Aspire.Workload.Tests/Aspire.Workload.Tests.csproj

```diff
@@ -0,0 +1,56 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+
+    <RunTestsOnHelix>true</RunTestsOnHelix>
+    <TestUsingWorkloads>true</TestUsingWorkloads>
+    <InstallWorkloadForTesting>true</InstallWorkloadForTesting>
```

Comment:
```suggestion
```

I don't think this is needed, since it defaults to `TestUsingWorkloads`

Reply:
It doesn't default to `TestUsingWorkloads`.

The workload testing targets install two sdks - `dotnet-latest`, and `dotnet-none`. The former has the manifest+workload installed, but that latter has only the manifest installed. It is useful for running tests that want to check the case the user does not have the workload installed.

The `TestUsingWorkloads` property essentially controls that. I'll open a PR in `dotnet/runtime` to rename this property so its purpose is clearer. And possibly clean this up a bit.

Reply:
Tracking in https://github.com/dotnet/aspire/issues/4222 .